.github/workflows/pr-check.yaml

```diff
@@ -200,6 +200,9 @@ jobs:
       - name: Run formatter
         run: yarn format:check
 
+      - name: Run build
+        run: yarn build
+
```

Comment:
@odockal  with https://github.com/containers/podman-desktop/pull/2757 merged you should be able to remove this